identity/selector/handler.go

```diff
@@ -71,15 +77,26 @@ func (h *handler) UseOrCreate(address, passphrase string) (id identity.Identity,
 func (h *handler) useExisting(address, passphrase string) (id identity.Identity, err error) {
 	id, err = h.manager.GetIdentity(address)
 	if err != nil {
-		return
+		return id, err
 	}
 
 	if err = h.manager.Unlock(id.Address, passphrase); err != nil {
-		return
+		return id, errors.Wrap(err, "failed to unlock identity")
+	}
+
+	registered, err := h.registry.IdentityExists(id, h.signerFactory(id))
+	if err != nil {
+		return id, errors.Wrap(err, "failed to verify registration status of local identity")
+	}
+	if !registered {
+		log.Info("existing identity is not registered, attempting to register")
+		if err = h.registry.RegisterIdentity(id, h.signerFactory(id)); err != nil {
```

Comment:
@zolia maybe it's time to drop identity registration at all - now it's a feature  identity stats collecting.
For Ethereum flow we can't just register identity, it's user flow, calling. For this flow user he will need to transfer money.

Reply:
Actually we a calling `registry.RegisterIdentity()` just to make `SetPayoutInfo()` to be successful, then user calls it later.

Lets remake SetPayoutInfo(), so that it fixes it's DB records